types.go

```diff
@@ -42,47 +52,52 @@ var (
 //      T *Thingy `optional:"true"`
 //    }
 //
-type In struct{}
+type In struct{ digSentinel }
```

Comment:
Ah, I thought you've been busy playing No Man's Sky.
I think the terminology is appropriate here.